csrc/fusion.h

```diff
@@ -95,6 +95,12 @@ enum class AllocationType : int {
   // input.  In this case, we use `ExpressionEvaluator` (instead of a kernel) to
   // cheaply compute the output tensor.
   PointerArithmetic,
+  // To evaluate outputs which are not aliases.
+  // TODO: This can be potentially merged with PointerArithmetic.
```

Comment:
```suggestion
  // TODO: This will be merged with PointerArithmetic in the next PR.
```

I'd clean this up sooner than later. Personally, I'd do this in the next PR, so we don't keep a easy-to-fix tech debt for too long in the codebase. 

See also: https://github.com/NVIDIA/Fuser/pull/1732#discussion_r1480853889

Reply:
Thanks for this note. On first pass I was wondering what is the difference since both signify that the output will be evaluated with `ExpressionEvaluator`.